src/libstd/sync/mutex.rs

```diff
@@ -393,6 +393,19 @@ impl<T: ?Sized + fmt::Debug> fmt::Debug for Mutex<T> {
     }
 }
 
+#[stable(feature = "std_guard_impls", since = "1.20")]
+impl<T: ?Sized + fmt::Display> fmt::Display for Mutex< T> {
```

Comment:
We tend to prefer `Display` for only values that can be losslessy displayed, so perhaps this could be removed?